parameter_files/WebParameters_2dii.yml

```diff
@@ -1,8 +1,8 @@
 default:
     paths:
-        project_location_ext: /Users/jacobkastl/Documents/git/PACTA_analysis/
-        data_location_ext: /Users/jacobkastl/Documents/git/pacta-data/2019Q4/
-        template_location: /Users/jacobkastl/Documents/git/create_interactive_report/
+        project_location_ext: ~/git/PACTA_analysis/
```

Comment:
I like the idea but I think this comes with the problem that it forces the user to have all repos in a directory called git. While I am personally happy with that, I am not sure everyone working on this has the same setup

Reply:
Yeah, here I was hard-wiring my own path. I was tempted to do something like this but suspected it wouldn't work:

```yaml
        project_location_ext: PACTA_analysis
        data_location_ext: pacta-data/2019Q4
        template_location: create_interactive_report
        project_location_ext: PACTA_analysis
```
But the code could be adapted so that it does work, for example, consider this pseudocode:

```r
project_location_ext_child_path <- config::get(file = "WebParameters_2dii.yml")$project_location_ext
project_location_ext <- here::here(project_location_ext_child_path)
```